source/frontend/src/features/properties/map/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -67,76 +69,93 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
-              </SectionField>
-              <SectionField label="Cost type" labelWidth={'4'}>
-                {acquisitionFile.project?.costTypeCode?.code ?? ''}
-              </SectionField>
-              <SectionField label="Fiscal year" labelWidth="4" contentWidth="4" required>
-                <Select field="fiscalYear" options={fiscalYearOptions} placeholder="Select..." />
-              </SectionField>
-            </Section>
-            <Section>
-              <SectionField label="Detailed remarks" labelWidth="12">
-                <MediumTextArea field="detailedRemarks" />
-              </SectionField>
-            </Section>
+              <Section header="Financial coding">
+                <SectionField label="Product" labelWidth="7">
+                  {acquisitionFile.product?.code ?? ''}
+                </SectionField>
+                <SectionField label="Business function" labelWidth="7">
+                  {acquisitionFile.project?.businessFunctionCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Work activity" labelWidth="7">
+                  {acquisitionFile.project?.workActivityCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Cost type" labelWidth="7">
+                  {acquisitionFile.project?.costTypeCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Fiscal year" labelWidth="7" contentWidth="4" required>
+                  <Select field="fiscalYear" options={fiscalYearOptions} placeholder="Select..." />
+                </SectionField>
+              </Section>
+
+              <Section header="Payment">
+                <SectionField label="Payee" labelWidth="3" required>
+                  <Select
+                    field="payee"
+                    options={payeeOptions.map<SelectOption>(x => {
+                      return { label: x.text, value: x.value };
+                    })}
+                    placeholder="Select..."
+                  />
+                </SectionField>
+              </Section>
+
+              <Section>
+                <SectionField label="Detailed remarks" labelWidth="12">
+                  <MediumTextArea field="detailedRemarks" />
+                </SectionField>
+              </Section>
+            </StyledContent>
 
             <Prompt
               when={formikProps.dirty}
               message="You have made changes on this form. Do you wish to leave without saving?"
             />
-            <SidebarFooter
-              onSave={() => formikProps.submitForm()}
-              isOkDisabled={formikProps.isSubmitting || !formikProps.dirty}
-              onCancel={() => cancelFunc(formikProps.resetForm, formikProps.dirty)}
-            />
+            <StyledFooter>
+              <SidebarFooter
+                onSave={() => formikProps.submitForm()}
+                isOkDisabled={formikProps.isSubmitting || !formikProps.dirty}
```

Comment:
Right, like if you just change the payee option can you even save since the formik form won't be considered dirty since it won't have registered any changes?

Reply:
it will as I mentioned in the previous comment. Let me know to jump on call to chat about this further